plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/catalog/jdbc/IcebergJdbcCatalogConfig.java

```diff
@@ -24,12 +24,27 @@
 
 public class IcebergJdbcCatalogConfig
 {
+    private String driverClass;
```

Comment:
nit: maybe allow not setting the driver if url is using `jdbc:postgresql` schema

Reply:
It may cause a minor issue if third-party JDBC developers re-use the prefix. I suppose CData uses the prefix with the different driver class `cdata.jdbc.postgresql.PostgreSQLDriver`. Let me keep as-is.